.github/workflows/jdk18.yml

```
@@ -0,0 +1,39 @@
+name: JDK18
+
+on:
+  push:
+    branches:
+      - '**'
+    paths:
+      - 'Casks/zulu-jdk18.rb'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'Casks/zulu-jdk18.rb'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: macos-latest
+    steps:
+
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: brew pull & reset & tap
+        run: |
+          brew update-reset "$(brew --repository)"
+          brew update-reset "$(brew --repository homebrew/cask)"
+          mkdir -p $(brew --repo)/Library/Taps/mdogan
+          ln -s $GITHUB_WORKSPACE $(brew --repo)/Library/Taps/mdogan/homebrew-zulu
+        env:
+          HOMEBREW_COLOR: 1
+          HOMEBREW_DEVELOPER: 1
+
+      - name: install jdk18
+        run: brew install --cask zulu-jdk18
```

```suggestion
        run: brew install zulu-jdk18
```

No need to add `--cask` now.